indra/llui/llcombobox.cpp

```diff
@@ -142,14 +141,11 @@ LLComboBox::LLComboBox(const LLComboBox::Params& p)
 	// Grab the mouse-up event and make sure the button state is correct
 	mList->setMouseUpCallback(boost::bind(&LLComboBox::onListMouseUp, this));
 
-	for (LLInitParam::ParamIterator<ItemParams>::const_iterator it = p.items.begin();
-		it != p.items.end();
-		++it)
+	for (LLComboBox::ItemParams item_params : p.items)
```

Comment:
Probably should be a const reference instead of a copy

Reply:
> Probably should be a const reference instead of a copy

Not in this case:
![image](https://github.com/secondlife/viewer/assets/124201357/16423b33-0c0a-4223-9baa-92336cbbbcee)